tests/api/patrons/test_patrons_tasks.py

```
@@ -0,0 +1,316 @@
+    _restore_patron_expiration(patron_martigny)
+    _make_user_recently_active(patron_martigny.user)
+
+
+def test_dry_run_does_not_delete(app, org_martigny, patron_martigny):
+    """Test that dry_run mode logs candidates without deleting."""
+    config = {"expiration_years": 3, "inactivity_years": 3}
+    _set_patron_cleanup_config(org_martigny, config)
+    _make_patron_expired(patron_martigny)
+    _make_user_inactive(patron_martigny.user)
+
+    deleted, skipped = _delete_inactive_patrons_for_org(org_martigny, config, dry_run=True)
```

Ruff (0.15.12)

[warning] Unpacked variable `deleted` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)